keda.advisories.yaml

```diff
@@ -13,3 +13,24 @@ advisories:
         data:
           type: vulnerable-code-not-included-in-package
           note: Go vulndb has marked this code NOT_IMPORTABLE.
+
+  - id: CVE-2023-39325
```

Comment:
is this one missing # CVE-2023-45142?

Reply:
I think you're right. CVE-2023-45142 wasn't found on the latest version of keda that we scanned